API.yaml

```diff
@@ -1,548 +1,670 @@
-      produces:
-      - "application/json"
+      - Analytics
+      summary: Save the operational data sent by the iOS device, in compliance with
+        the monthly policy, authorized with the provided analytics token.
       parameters:
-      - in: "header"
-        name: "Immuni-Dummy-Data"
-        type: "integer"
-        required: true
-        minimum: 0
-        maximum: 1
-      - in: "body"
-        name: "body"
-        description: "A JSON-formatted dictionary containing the operational data."
+      - name: Immuni-Dummy-Data
+        in: header
         required: true
         schema:
-          $ref: "#/definitions/OperationalInfo"
+          maximum: 1
+          minimum: 0
+          type: integer
+          format: int32
+      requestBody:
+        description: A JSON-formatted dictionary containing the operational data.
+        content:
+          application/json; charset=utf-8:
+            schema:
+              $ref: '#/components/schemas/OperationalInfo'
+        required: true
       responses:
         "204":
-          description: "Operation completed successfully."
+          description: Operation completed successfully.
           headers:
             Cache-Control:
-              type: "string"
-              description: "Directives to prevent caching of 204 responses."
+              description: Directives to prevent caching of 204 responses.
+              schema:
+                type: string
+          content: {}
         "400":
-          description: "Invalid data. An APIError is returned."
-          schema:
-            $ref: "#/definitions/APIError"
+          description: Invalid data. An APIError is returned.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/APIError'
       security:
-        - analytics_bearer: []
+      - analytics_bearer: []
+      x-codegen-request-body-name: body
   /analytics/google/operational-info:
     post:
       tags:
-      - "Analytics"
-      summary: "Save the operational data sent by the Android device, authorized with the provided SafetyNet hardware attestation token."
-      consumes:
-      - "application/json; charset=utf-8"
-      produces:
-      - "application/json"
+      - Analytics
+      summary: >-
+        Save the operational data sent by the Android device, authorized with
+        the provided SafetyNet hardware attestation token.
       parameters:
-      - in: "header"
-        name: "Immuni-Dummy-Data"
-        type: "integer"
-        required: true
-        minimum: 0
-        maximum: 1
-      - in: "body"
-        name: "body"
-        description: "A JSON-formatted dictionary containing the operational data."
+      - name: Immuni-Dummy-Data
+        in: header
         required: true
         schema:
-          $ref: "#/definitions/GoogleOperationalInfo"
+          maximum: 1
+          minimum: 0
+          type: integer
+          format: int32
```

Comment:
I was wondering whether it's worth creating a `#/components/schemas/ImmuniDummyData` since reusable in four places.

Reply:
I wanted to do it after the conversion, so that we always scope PR goals.

Reply:
Yes yes, that's great! I wanted to understand whether you thought it was worth it. Please, go ahead.